src/auth/LoginProvider.tsx

```diff
@@ -137,17 +145,15 @@ const LoginProvider = ({
   const onActive = useCallback(
     throttle(
       () => {
-        const stytchClient = authState.client;
-        if (!stytchClient) return;
-
-        const session = stytchClient.session.getSync();
+        if (!isAuthStytch()) return;
 
-        if (session) {
-          stytchClient.session.authenticate({
+        if (authState.client.session.getSync()) {
+          authState.client.session.authenticate({
             session_duration_minutes: 1440
           });
         } else if (hasAuthedRef.current) {
-          stytchClient.session.revoke().then(logoutActions);
+          // There is no session, so need to revoke it
```

Comment:
I think this change will fix the sentry error about user needing to be authenticated